scripts/generate-style-code.js

```diff
@@ -6,9 +6,6 @@ const ejs = require('ejs');
 const spec = require('./style-spec');
 const colorParser = require('csscolorparser');
 
-// FIXME: https://github.com/mapbox/mapbox-gl-native/issues/15008
-delete spec.layout_circle["circle-sort-key"]
```

Comment:
Also remove these lines from the Darwin version of this script:

https://github.com/mapbox/mapbox-gl-native/blob/1f266b45bc1cf5bec43cbd81a2da587023124401/platform/darwin/scripts/generate-style-code.js#L18-L19

Reply:
In the future, these properties should be deleted in scripts/style-spec.js instead of both scripts/generate-style-code.js and platform/darwin/scripts/generate-style-code.js. This is the reason for the layout of indirection in scripts/style-spec.js.

Reply:
Now that most of platform/darwin/ has been deleted in favor of the sources in the mapbox/mapbox-gl-native-ios repository, mapbox/mapbox-gl-native-ios#272 will take care of deleting the remnants of this override.